dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerAPIImpl.java

```diff
@@ -353,18 +347,62 @@ public void saveContainerStructures(final List<ContainerStructure> containerStru
     @Override
     public List<ContentType> getContentTypesInContainer(Container container) throws DotStateException, DotDataException, DotSecurityException  {
 
-        List<ContainerStructure> csList =  getContainerStructures(container);
-        List<ContentType> cTypeList = new ArrayList<ContentType>();
+		try {
+			List<ContainerStructure>  csList = getContainerStructures(container);
 
-        for (ContainerStructure cs : csList) {
-            ContentType type = APILocator.getContentTypeAPI(APILocator.systemUser()).find(cs.getStructureId());
-            if(type==null) continue;
-            cTypeList.add(type);
-        }
-        
-        return cTypeList.stream().sorted(Comparator.comparing(ContentType::name)).collect(Collectors.toList());
+			List<ContentType> cTypeList = new ArrayList<>();
+
+			for (ContainerStructure cs : csList) {
+				try {
+					ContentType type = APILocator.getContentTypeAPI(APILocator.systemUser()).find(cs.getStructureId());
+					if(type == null) continue;

```

Comment:
avoid using if without curly bracket, must be such as 

if (null == type) {
   continue;
}